releases/latest/mysql-bundle.yaml

```diff
@@ -1,29 +1,142 @@
 applications:
+  data-integrator:
+    channel: latest/edge
+    charm: data-integrator
+    constraints: arch=amd64
+    num_units: 1
+    revision: 27
+    series: jammy
+    to:
+    - '0'
+  grafana-agent:
+    channel: latest/edge
```

Comment:
question: do we want to test against all things `edge` or only the ones under the team umbrella?

Reply:
i will merge as is, as postgresql has `edge` for charms not in the data platform umbrella. however, i will confirm with @taurus-forever and open a follow up PR if necessary

Reply:
alex's answer: we should use edge for now, but there will be more changes upcoming to this bundle in the near future